docs/_data/paths/rating-proposals_forecasts.yaml

```diff
@@ -146,8 +146,9 @@ patch:
       application/vnd.trolie.rating-forecast-proposal.v1+json:
         schema:
           $ref: ../components/schemas/forecast-rating-proposal.yaml
-        example:
-          $ref: ../examples/forecast-ratings-proposal-patch.json
+        examples:
+          simple-example:
+            externalValue: '/example-narratives/example-json/forecast-ratings-proposal-patch.json'
```

Comment:
I like this best, but there was at least one tool (an examples validator) which would not follow this reference. We're not going to use that tool--Redocly validates examples and generates the doc viewer just fine--but we might want to use a few code generators to make sure they don't choke.

We can use `examples` with inline or `externalValue`, of course.

Reply:
One downside is that the generated `openapi.yaml` is not self-contained, as the references will always live at, e.g., `trolie.energy/example-narratives/example-json/forecast-ratings-proposal-patch.json`

Reply:
It turns out [Redocly doesn't dereference externalValue examples](https://github.com/Redocly/redocly-cli/issues/919), so we'd lose our example validation from `redocly lint`. 

Anyway, a (normal ref works)[https://github.com/trolie/spec/blob/5db28341bb0a470ca2abc8d39406fd567a38507c/docs/_data/paths/limits_forecast-snapshot.yaml#L45], since the example just gets inlined into the `openapi.yaml` by the `redocly bundle`. This seems to have the least number of drawbacks, and I think it's the approach we should use.

Reply:
I agree, will follow.